src/__test__/components/Header/Header.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Header from '@components/Header';
+import { render } from '@testing-library/react';
+import * as SidebarContext from '@store/sidebarContext';
+
+describe('Header component', () => {
+  it('should be able to render the Header component', () => {
+    const { getByText, getByPlaceholderText, getByTestId } = render(
```

Comment:
Since `testing-library `give for as a object that can list all the functions to use in a test, I think it is better to use screen instead of create a destructor object from render